static_analysis/Dockerfile

```diff
@@ -0,0 +1,86 @@
+#######
+# cppcheck build
+######
+
+FROM alpine:latest AS cppcheck_build
+
+# Version 1.90
+ARG cppcheck_commit=077e652
+
+RUN \
```

Comment:
Doesn't this (empty line after `RUN`) generate a warning when building? It definitely used to.

Reply:
No warning, but it's wrong and a hangover from a restructuring I did mid-way